SparkPost/2web2ui
# FE-437: Track API error events

**How to test?**
1. Open Google Tag Manager and click "Preview"
1. For SparkPost API errors, try to [open a webhook that doesn't exist](http://app.sparkpost.test/webhooks/details/1867aae0-39ae-11e8-9d73-1505f6674400?subaccount=0)
1. For Zuora API errors, try to update payment informaiton, but only enter part of the credit card number.
1. Both events should appear in the GTM preview console
